subprojects/docs/src/samples/userguide/dependencies/disablingTransitiveDependencyResolution/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'deactivating-transitive-dependency-resolution'
```

Comment:
Is it an issue that the project name doesn't really match the directory? "deactivating" vs. "disabling"

Reply:
It's not an issue. It's more what the subdirectory was called in the process when I developed it. I changed it to "disabling".

Reply:
👍